File: card.rb
class Card

    attr_reader :number, :suit
    
    def initialize(number, suit)
        @number = number
        @suit = suit
    end

    def to_s()
        prefix = ''
        if @number < 10 
            prefix = ' ' 
        end
        prefix + @number.to_s + @suit
    end

end

File: card.py
from suit import Suit

class Card:

    def __init__(self, value: Suit, suit: Suit):
        self.value = value
        self.suit = suit

    def __str__(self):
        prefix = " " if self.value < 10 else ""

        return  f"{prefix}{self.value}{self.suit.value}"
